.github/workflows/post-merge-internal-api.yml

```diff
@@ -6,9 +6,13 @@ on:
             - master
             - 'release-hotfix-*'
         paths:
-            - 'src/Aquifer.Common/**'
+            - 'Directory.Build.*'
```

Comment:
Can just be `Directory.Build.props` instead of wildcard. Also, we need the new `Directory.Packages.props` file included.

Reply:
Oops I misread and thought the other was `Directory.Build.packages`